build_webpack/webpack.terminal.config.js

```diff
@@ -2,11 +2,26 @@ const webpack = require('webpack');
 const base = require('./webpack.base.config');
 const merge = require('webpack-merge');
 const nodeExternals = require('webpack-node-externals');
+const addWithsConfig = require('../scripts/npm/with');
+
+const default_arguments = {
```

Comment:
How are the default_arguments handled? Are they overwritten by the command line arguments?

Reply:
Yes.

If you got a key in default_arguments, it gets overwritten by the arguments passed in with's command line.